pkg/datapath/linux/node.go

```diff
@@ -1449,7 +1449,7 @@ func (n *linuxNodeHandler) NodeConfigurationChanged(newConfig datapath.LocalNode
 	prevConfig := n.nodeConfig
 	n.nodeConfig = newConfig
 
-	if n.nodeConfig.EnableIPv4 {
+	if n.nodeConfig.EnableIPv4 || n.nodeConfig.EnableIPv6 {
```

Comment:
Is there even a case where both ipv4 and ipv6 are disabled? Seems we can always have this on and skip check?

Reply:
I think not, I can remove the check in a follow-up, liked the explicit check, but either way is fine.